src/Features/ERDDAP/waterLevel/TidesTable.tsx

```diff
@@ -0,0 +1,74 @@
+import { PlatformFeature, Tides } from "Features/ERDDAP/types"
+import { useUnitSystem } from "Features/Units"
+import { converter } from "Features/Units/Converter"
+import { getValueWithOffset } from "Features/Units/Converter/data_types/_tidal_level"
+import { useEffect, useState } from "react"
+import { Table } from "reactstrap"
+import { PREDICTED_WATER_LEVEL_STANDARDS, WATER_LEVEL_STANDARDS } from "Shared/constants/standards"
+import { round } from "Shared/math"
+
+interface TidesTableProps {
+  platform: PlatformFeature
+  standardName: string
+  /** Datum offset BEFORE unit conversion */
+  datumOffset: number
+}
+
+export const TidesTable = ({ platform, standardName, datumOffset }: TidesTableProps) => {
+  const [nextTides, setNextTides] = useState<Tides[]>()
+  const unitSystem = useUnitSystem()
+  const dataConverter = converter(standardName)
+
+  useEffect(() => {
+    const futureTides = platform.properties.readings.find(
+      (ts) => ts.type === "Prediction" && PREDICTED_WATER_LEVEL_STANDARDS.includes(ts.data_type.standard_name),
```

Comment:
I think I would just use a single `WATER_LEVEL_STANDARDS` and look for `ts.type === "Prediction"` or `ts.type === "Forecast"` (I swear, I've almost got Hydromet tabular forecasts working...), as there isn't actually a 'predicted' standard_name, it's just one that we've added.

Here are the ones I'm using for Buoy Barn right now https://github.com/gulfofmaine/buoy_barn/blob/7c56d1cb9c8710e64c2396f7acb76d1989b2c5d0/app/deployments/standard_names.py#L1-L6